Makefile

```diff
@@ -202,7 +202,7 @@ deploy: ensure-kustomize install
 	mkdir -p overlays/deploy
 	cp overlays/template/kustomization.yaml overlays/deploy
 	cd overlays/deploy && ../../$(KUSTOMIZE) edit set image registry.ci.openshift.org/openshift/hive-v4.0:hive=${IMG} && ../../$(KUSTOMIZE) edit set namespace ${HIVE_OPERATOR_NS}
-	$(KUSTOMIZE) build overlays/deploy | oc apply -f -
+	$(KUSTOMIZE) build overlays/deploy | sed 's/        - info/        - debug/' | oc apply -f -
```

Comment:
I wish there was a more robust way to do this, but it's probably good enough as is

Reply:
Yeah. I looked into using kustomize, but its `edit` subcommand is super limited. I did verify that this was the only hit for `info` (never mind the spaces and hyphen) in the whole manifest.